spring-examples/spring-worker-example/src/main/java/io/vertx/example/spring/worker/Application.java

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
More of a generic question here, should we copyright the examples? the repo is inconsistent